rfcs/rfcOVID-0018.md

```diff
@@ -0,0 +1,104 @@
+# A feature to automatically "yield true" at the end of a file
+
+## Preamble
+
+    Author:  Curtis "Ovid" Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      OVID-0018
+    Status:  Draft
+
+## Abstract
+
+This RFC proposes a `yield_true` feature. When used, the current Perl _file_
```

Comment:
I'd avoid naming the feature so early; as we'll see later on the name has issues. Perhaps just put

    This RFC proposes a feature which, when used, causes the current Perl file to ...